src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/renderGenericComponent.js

```diff
@@ -73,6 +84,28 @@ function renderGiftCard(paymentMethod) {
   document.querySelector('#giftCardLabel').classList.remove('invisible');
 }
 
+function applyGiftCard() {
+  const now = new Date().toISOString();
+  const { amount } = store.checkoutConfiguration;
+  const { orderAmount, expiresAt } = store.partialPaymentsOrderObj;
+
+  const isPartialPaymentExpired = expiresAt && now > expiresAt;
+  const isCartModified =
```

Comment:
Can I modify this with the right breakpoint?

Reply:
it would be possible to manipulate the code with breakpoints but that doesn't constitute as a security vulnerability. Also I think SFCC code when deployed would be bundled and not as easily readable.

Reply:
Anything I should do about this?